src/robots.txt

```diff
@@ -1 +1,15 @@
-# All robots welcome!
\ No newline at end of file
+---
+layout: none
+---
+
+{% comment %}
+
+This is the default robots file used when serving locally or on staging servers.
+Instructions for deploying to the site's official server are in the README.md.
+
+{% endcomment -%}
+
+User-agent: *
+Disallow: /
+User-agent: linkcheck
+Allow: /
```

Comment:
Note that linkcheck hasn't yet been updated to recognize this entry (cc @filiph).